osmosis-installer.py

```diff
@@ -561,7 +609,7 @@ def initSetup ():
         print(bcolors.OKGREEN + "(3/4) Installing Go..." + bcolors.ENDC)
         subprocess.run(["wget -q -O - https://git.io/vQhTU | bash -s -- --version 1.17.2"], shell=True, env=my_env)
         #print(bcolors.OKGREEN + "Reloading Profile..." + bcolors.ENDC)
-        #subprocess.run([". "+ HOME+"/.profile"], shell=True)
+        #subprocess.run([". "+HOME+"/.profile"], shell=True)

```

Comment:
Is there a reason for keeping dead code?

Reply:
No, you're right I will take this out. Python is really tricky with the shell since you actually cant reload the profile in the same script, so you have to manually set an env variable and attach it to the subprocess.